tests/test_ledger.py

```diff
@@ -232,3 +259,66 @@ def test_delete_non_existent_ledger():
     cashctrl_ledger = CashCtrlLedger()
     with pytest.raises(RequestException):
         cashctrl_ledger.delete_ledger_entry(ids='non-existent')
+
+def test_mirror_ledger(add_vat_code):
+    cashctrl_ledger = CashCtrlLedger()
+    target = StandaloneLedger.standardize_ledger(pd.concat([
+        individual_transaction,
+        collective_transaction,
+    ]))
+
+    # Mirror with one single and one collective transaction
+    cashctrl_ledger.mirror_ledger(target=target)
+    mirrored = cashctrl_ledger.ledger()
+    assert txn_to_str(target) == txn_to_str(mirrored)
+
+    # Mirror with duplicate transactions and delete=False
+    new_individual_transaction = individual_transaction.copy()
+    new_individual_transaction.at[0, 'id'] = 5;
+    new_collective_transaction = collective_transaction.copy()
+    new_collective_transaction.at[0, 'id'] = 6;
+    new_collective_transaction.at[1, 'id'] = 6;
+    target = StandaloneLedger.standardize_ledger(pd.concat([
+        individual_transaction,
+        new_individual_transaction,
+        collective_transaction,
+        new_collective_transaction,
+    ]))
+    cashctrl_ledger.mirror_ledger(target=target)
+    mirrored = cashctrl_ledger.ledger()
```

Comment:
Because I do this in the `txn_to_srt()` function [line 65](https://github.com/macxred/cashctrl_ledger/blob/2edb906e97472bc055004b692c933c22e424c40f/tests/test_ledger.py#L64)

```python
def txn_to_str(df: pd.DataFrame) -> List[str]:
    df = df.drop(columns=['id']).reset_index(drop=True)
```